src/components/Form/Form.jsx

```diff
@@ -1,24 +1,41 @@
 import React from 'react';
-import './Form.css';
 import { Link } from 'react-router-dom';
+import './Form.css';
 
 const Form = () => {
   return (
     <div className='Background_Form'>
-      <h1 className='Trott_Logo'>TROTT</h1>
-      <p className='Enter_Number'>Entre ton numéro de téléphone</p>
-      <p className='Text_Form'>Puis rentre ton code de confirmation</p>
-      <input type="tel"
-        className="Input_tel"
-        pattern="[0-9]{2}-[0-9]{2}-[0-9]{2}"
-        required />
-      <div>
-        <Link className='Without_Connect' to="/sms_de_verification">Suivant</Link>
+      <img
+        src="https://cdn.discordapp.com/attachments/714501146701463636/720283550003363880/TROTT.png"
+        className="logo"
+        alt="logo"
+      />
+      <div className="Text_Container">
+        <p className='Enter_Number'>Entre ton numéro de téléphone</p>
+        <p className='Text_Form'>Puis rentre ton code de confirmation
+          <img
+            src="https://images.emojiterra.com/google/android-10/512px/1f609.png"
+            className="smiley"
+            alt="test"
+          />
+        </p>
+      </div>
+      <div className="Input_Container">
+        <input type="number" className="Input_Phone" placeholder='Number Phone' />
+        <hr className='hr' />
```

Comment:
looks more like a border in your div

Reply:
you prefer to put a border instead of an "hr" ?

Reply:
hr is html: it's content. moreover hr has a specific semantic: it's a thematic separation inside a container, so there is a content before and a content after the thematic separation. there is nothing after your hr, so it's not a separation between contents thus it's not a hr ;)

your code here looks like style: this should be handled with css